### PR TITLE
chore: watch mode first cut

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: examples/todomvc/tests/integration.spec.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ const TODO_ITEMS = [
 
 test.describe('New Todo', () => {
   test('should allow me to add todo items', async ({ page }) => {
+    expect(1).toBe(2);
```
Comment (mxschmitt) [inline examples/todomvc/tests/integration.spec.ts:21]: revert this

File: examples/todomvc/tests/integration.spec.ts [hunk 2]
```diff
@@ -44,6 +45,7 @@ test.describe('New Todo', () => {
   });
 
   test('should clear text input field when an item is added', async ({ page }) => {
+    expect(1).toBe(2);
```
Comment (mxschmitt) [inline examples/todomvc/tests/integration.spec.ts:48]: ditto

File: packages/playwright-test/package.json [hunk 3]
```diff
@@ -33,6 +33,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@types/node": "*",
+    "chokidar": "3.5.3",
```
Comment (mxschmitt) [inline packages/playwright-test/package.json:36]: I think instead we want to say: ^3.5 to not break the ecosystem when other tools like vite also already depend on it.

File: packages/playwright-test/src/cli.ts [hunk 4]
```diff
@@ -175,7 +169,9 @@ async function runTests(args: string[], opts: { [key: string]: any }) {
   config._internal.passWithNoTests = !!opts.passWithNoTests;
 
   const runner = new Runner(config);
-  const status = await runner.runAllTests();
+  if (opts.watch)
+    process.stdout.write('\x1Bc');
```
Comment (mxschmitt) [inline packages/playwright-test/src/cli.ts:173]:
```suggestion
    process.stdout.write('\x1Bc'); // Reset the terminal
```

Not sure if its worth mentioning with the other commands.

File: packages/playwright-test/package.json (hunk 3 above)
Comment (pavelfeldman) [reply packages/playwright-test/package.json:36]: I am actually thinking the opposite, I'd put it into bundle if I could.

**Review by dgozman — APPROVED** [review]

**Pull request merged**